src/components/DocumentationTopic.vue

```diff
@@ -11,9 +11,9 @@
 <template>
   <div class="doc-topic">
     <main class="main" id="main" role="main" tabindex="0">
-      <DocumentationHero :type="symbolKind || role">
+      <DocumentationHero :type="symbolKind || role" :enableHero="enableHero">
```

Comment:
This isn't a huge deal, but we should consider renaming this to better convey what it does—this flag simply adds/removes the special dark background for the hero as opposed to actually enabling/disabling it.